xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm

```diff
@@ -2216,19 +2217,19 @@ $displayName##
   #set($parameters.firstItem = $parameters.crtPage * $parameters.itemsPerPage)
   #set($parameters.lastPage = $parameters.pageCount - 1)
   <div class="paginationFilter">
-    #if($parameters.position == 'top')
-      #paginationResultsRange($parameters)
-    #end
     #if($parameters.totalItems > 0)
       #if($parameters.controlsAround == true)
-        #paginationControlLeft($parameters)
-        #paginationPages($parameters)
-        #paginationControlRight($parameters)
+        #_paginationControlLeft($parameters)
+        #_paginationPages($parameters)
+        #_paginationControlRight($parameters)
       #else
-        #paginationControl($parameters)
-        #paginationPages($parameters)
+        #_paginationPages($parameters)
+        #_paginationControl($parameters)
       #end
     #end
+    #if($parameters.position == 'top')
+      #paginationResultsRange($parameters)
```

Comment:
How come this is not prefixed with underscore? How is this macro different from ``_paginationPages`` for instance?